game/script-topics.rpy

```diff
@@ -1044,61 +1044,238 @@ init 5 python:
+
+            if len(activity_options) > 0:
+                $ activity_mention = random.choice(activity_options)
+                n 2ccsfl "I mean...{w=1}{nw}"
+                extend 2ulraj " I know you use your computer already for a bunch of stuff.\n{w=0.75}{nw}"
+                extend 4fchbg "Like [activity_mention]!"
+
+        else:
+            n 2ccsfl "I mean...{w=1}{nw}"
+            extend 2tllsl " I guess you probably still use your computer for other stuff already."
+
+        n 1ccsflsbl "But that still doesn't change the fact you're gonna be racking up a load of extra screen time thanks to me."
+        
+        if Natsuki.isEnamored(higher=True):
+            n 2clrfllsbl "And the last thing I wanna hear about is you getting yourself all cramped up because you were slouched over like a potato for hours."
+            n 2cllsl "..."
+
+        else:
+            n 2ccsaj "And the last thing I wanna hear is you complaining your back hurts because nobody had the brains to tell you not to slouch like a potato for hours."
+            n 2ccspol "I-{w=0.2}I'm not having {i}that{/i} on my conscience."
+
+        n 1ccsss "Heh.{w=0.75}{nw}"
+        extend 4fsrss " In fact..."
+        n 2fnmbg "You know what,{w=0.2} [player]?{w=0.75}{nw}"
+        extend 2fsqsm " I hope you're sitting."
+        n 6fcsbg "'Cause I'm gonna make sure you're using your computer the {i}right{/i} way,{w=0.5}{nw}"
+        extend 3fchgn " whether you like it or not!"
+
+    n 1fcsbg "Alright,{w=0.75}{nw}"
+    extend 6fcsaj " so number one:{w=1}{nw}" 
+    extend 3unmbg " posture!"
+    n 1ccsflsbl "...And no,{w=0.2} [player].{w=0.75}{nw}"
+    extend 4csqtr " I mean {i}actual{/i} posture.{w=0.75}{nw}"
+    extend 4ccspo " The sitting kind."
+    n 3fnmfl "Now sit up straight,{w=0.2} and keep that back of yours against the chair,{w=0.2} [player].{w=0.75}{nw}"
+    extend 3fcsgs " I mean it!"
+    n 4cllfl "Seriously -{w=0.5}{nw}"
+    extend 4tsqem " unless you feel like making friends with your nearest chiropractor,{w=0.5}{nw}"
+    extend 4cslem " trust me when I say you {i}really{/i} don't wanna spend all your time slouching."
+    n 2csqss "...Or hunched up on your chair like some kind of weird computer gremlin."
+    n 6unmaj "If you're doing it right,{w=0.5}{nw}"
+    extend 3ullaj " then you should have your arms and thighs parallel to the ground,{w=0.5}{nw}"
+    extend 3fcssm " with your eyes roughly at the top of your screen.{w=0.75}{nw}"
+    extend 3fcsbg " Easy peasy!"
+    n 4unmfl "Oh,{w=0.2} right -{w=0.5}{nw}"
+    extend 2clrsssbl " make sure your feet can still touch the floor,{w=0.2} though.\n{w=0.75}{nw}"
+    extend 2fcscasbr "Even {i}I{/i} can do that,{w=0.2} [player]."
+    n 1fnmaj "Number two:{w=0.75}{nw}"
+    extend 2fcsbg " distance!"
+    n 2clrbg "It's pretty easy to forget,{w=0.5}{nw}"
+    extend 4tnmbo " but if you wanna avoid sore eyes then you've gotta make sure you're actually a decent distance from the screen too."
```

Comment:
Could maybe add to look out the window to let the eyes focus on larger distance than just the closed up desktop.

Reply:
Sure, I can add a segment for this